src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioTransformationServiceProcessAudioRequestTest.java

```diff
@@ -52,30 +64,26 @@ void processAudioRequestShouldSucceedAndUpdateRequestStatusToCompletedForDownloa
         var mediaRequestEntity = dartsDatabase.getMediaRequestRepository()
             .findById(mediaRequestId)
             .orElseThrow();
-
         assertEquals(COMPLETED, mediaRequestEntity.getStatus());
-    }
-
-    @Test
-    void processAudioRequestShouldSucceedAndUpdateRequestStatusToCompletedForPlaybackRequestType() {
-        given.databaseIsProvisionedForHappyPath();
-        given.aMediaRequestEntityForHearingWithRequestType(hearing, AudioRequestType.PLAYBACK);
-
-        Integer mediaRequestId = given.getMediaRequestEntity().getId();
-
-        UUID blobId = audioTransformationService.processAudioRequest(mediaRequestId);
-        assertNotNull(blobId);
 
-        var mediaRequestEntity = dartsDatabase.getMediaRequestRepository()
-            .findById(mediaRequestId)
-            .orElseThrow();
+        List<NotificationEntity> scheduledNotifications = dartsDatabase.getNotificationRepository()
+            .findAll();
+        assertEquals(1, scheduledNotifications.size());
 
-        assertEquals(COMPLETED, mediaRequestEntity.getStatus());
+        var notificationEntity = scheduledNotifications.get(0);
+        assertEquals(NOTIFICATION_TEMPLATE_ID_SUCCESS, notificationEntity.getEventId());
+        assertNull(notificationEntity.getTemplateValues());
+        assertEquals("OPEN", notificationEntity.getStatus());
```

Comment:
This all looks good to me @chrisbellingham-hmcts - it's a pity we don't have an enum for the NotificationEntity - String status
e.g. MediaRequestEntity - AudioRequestStatus status

Reply:
Ah turns out an enum does exist, it's just the `status` field in the entity doesn't use it. I'll raise a separate MR to update the entity and subsequent impact.

Reply:
https://github.com/hmcts/darts-api/pull/305